packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -218,6 +218,24 @@ export default class MediaEmbedEditing extends Plugin {
 							'</video>'
 						);
 					}
+				},
+				{
+					name: 'Gong',
+					url: [
+						/^app\.gong\.io\/call\?id=([\w-]+)/
+					],
+					html: match => {
+						const id = match[ 1 ];
+
+						return (
+							'<div style="position: relative; padding-bottom: 100%; height: 0; padding-bottom: 56.2493%;">' +
+								`<iframe src="https://app.gong.io/embedded-call?call-id=${ id }" ` +
+									'style="position: absolute; width: 100%; height: 100%; top: 0; left: 0;" ' +
+									'frameborder="0" allow="autoplay; encrypted-media" allowfullscreen>' +
```

Comment:
Don't really understand why there is autoplay, encrypted-media, and allowfullscreen here... doesn't apply since it's not a video? This doesn't look like the iframe that Quenton posted in slack